Survey/Configurations/Keychain/KeychainAccess.swift

```diff
@@ -22,10 +22,10 @@ enum KeychainAccess {
         get {
             let decoder = JSONDecoder()
             guard let savedToken = keychain[data: Keys.token.rawValue],
-                  let user = try? decoder.decode(KeychainToken.self, from: savedToken) else {
+                  let token = try? decoder.decode(KeychainToken.self, from: savedToken) else {
```

Comment:
I suggest you create a new init `init(service: String)` and dependency injection with TokenStoreUseCase so that you can test TokenStoreUseCase without impacting the real data.